### PR TITLE
docs: update description breadcrumb

### PR DESCRIPTION
**Pull request by linstedt**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Build (`yarn build`) was run locally and any changes were pushed
- [ ] Unit tests (`yarn test`) were run locally and passed
- [ ] Visual Regression Tests (`yarn visual-regression`) were run locally and passed
- [ ] Linting (`npm lint`) was run locally and passed

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bug fix
- [ ] Feature
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [x] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

no ux desription :) 

GitHub Issue Number: none

## What is the new behavior?

Introduced new breadcrumb ux description, A few open points need to be clarified before merge
- width and truncation of breadcrumb items: 
   - min and max width? 
   - truncation should be with ellipsis #397
   - add overflow illustration according to information
- interactivity of items: children can only be shown for last item? Mismatch with Figma component here
- states: what states are available for interactive and not interactive items? Mismatch with Figma component here

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Review by danielleroux — APPROVED** [review]
lgtm

File: packages/documentation/docs/controls/_breadcrumb_styleguide.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+### Interactive item
+
+| State   | Description                                                                          |
+| ------- | ------------------------------------------------------------------------------------ |
+| Hover   | The hover state is indicated by font and fill color change.                          |
+| Active  | The active state is indicated by font and fill color change. Equal to pressed state. |
+| Focused | The focused state is indicated by a focus frame.                                     |
+
+<!-- focused state on interactive item available? -->
```
Comment (silviowolf) [inline packages/documentation/docs/controls/_breadcrumb_styleguide.md:71]: Yes, focus state should be available on interactive element, is currently missing --> issue #417

File: packages/documentation/docs/controls/_breadcrumb_styleguide.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+### Not interactive item
+
+| State   | Description                                      |
+| ------- | ------------------------------------------------ |
+| Hover   | The hover state is indicated by ...              |
+| Active  | Not available.                                   |
+| Focused | The focused state is indicated by a focus frame. |
+
+<!-- focused state on not interactive item available? -->
```
Comment (silviowolf) [inline packages/documentation/docs/controls/_breadcrumb_styleguide.md:81]: - no focus on non-interactive "last" item

File: packages/documentation/docs/controls/_breadcrumb_styleguide.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+<!-- hover state not interactive item: bold in light theme, white in dark theme? -->
```
Comment (silviowolf) [inline packages/documentation/docs/controls/_breadcrumb_styleguide.md:82]: no, "last" item should not have an hover state nor other state change

File: packages/documentation/docs/controls/_breadcrumb_styleguide.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+
+| State   | Description                                                                          |
+| ------- | ------------------------------------------------------------------------------------ |
+| Hover   | The hover state is indicated by font and fill color change.                          |
+| Active  | The active state is indicated by font and fill color change. Equal to pressed state. |
+| Focused | The focused state is indicated by a focus frame.                                     |
+
+<!-- focused state on interactive item available? -->
+
+### Not interactive item
```
Comment (silviowolf) [inline packages/documentation/docs/controls/_breadcrumb_styleguide.md:73]: Chapter can be removed. There are no interactions on the non-interactive "last" item available.

File: packages/documentation/docs/controls/_breadcrumb_styleguide.md [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+### Main elements
+
+The breadcrumb consists of at least one item. Each item contains a breadcrumb label. All but the last item in the breadcrumb path are always followed by a chevron icon.
+
+### Sizing
+
+- The breadcrumb has a fix height defined by breadcrumb item height.
+- The breadcrumb has a maximum width defined by the content area width. Exceeding the width will trigger truncation.
+- The breadcrumb item has a fixed height for single-line text entries.
+- The breadcrumb item has a minimum width of ... and maximum width of .... Between these limits, the item width is depending on the content.
```
Comment (silviowolf) [inline packages/documentation/docs/controls/_breadcrumb_styleguide.md:95]: Breadcrumb item has no min-width.
I do not know, if a max-width is set. @nuke-ellington?
If no max-width is set, change the line to "The breadcrumb item width is depending on the content."

Comment (nuke-ellington) [reply packages/documentation/docs/controls/_breadcrumb_styleguide.md:95]: no max-width

File: packages/documentation/docs/controls/_breadcrumb_styleguide.md [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+Go to:
+
+- [Variants](#variants)
+- [Behavior in context](#behavior-in-context)
+- [States](#states)
+- [Dos and don't](#dos-and-don't)
+
+## Usage
+
+Breadcrumbs make the structure of your appliction transparent to the user. They offer means to navigate through the application.
```
Comment (kathrinschalber) [inline packages/documentation/docs/controls/_breadcrumb_styleguide.md:14]: appliction -> application

File: packages/documentation/docs/controls/_breadcrumb_styleguide.md [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+
+### Content
+
+Breadcrumb items can, but don't have to, include an icon. The icon is positioned before the breadcrumb label. Icons can be included for each item or only for specific items (e.g. the root item).
+
+## Behavior in context
+
+### Population
+
+The breadcrumb should be populated location-based to reflect the hierarchy of the application and the loction of the user in it. The current location should always be included in the breadcrumb.
```
Comment (kathrinschalber) [inline packages/documentation/docs/controls/_breadcrumb_styleguide.md:44]: loction -> location

File: packages/documentation/docs/controls/_breadcrumb_styleguide.md [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+- The breadcrumb item has a fixed height for single-line text entries.
+- The breadcrumb item width depends on the content and is limited to a maximum width of 14 rem.
+
+### Spacing
+
+Between breadcrumb items, a space of 0.25 rem is introduced.
+
+## Dos and don'ts
+
+- Don't use breadcrumbs to display a multistep process. Use a workflow control instead.
```
Comment (kathrinschalber) [inline packages/documentation/docs/controls/_breadcrumb_styleguide.md:93]: link the last sentence to the workflow chapter? (same for the releated patterns)

File: packages/documentation/docs/controls/_breadcrumb_styleguide.md [hunk 9]
```diff
@@ -0,0 +1,101 @@
+Breadcrumb navigation is a UI control that allows users to track their location within an application and easily navigate to previous or child pages.
```
Comment (kathrinschalber) [inline packages/documentation/docs/controls/_breadcrumb_styleguide.md:1]: I can navigate to children? :o

**Pull request merged**